pkgs/applications/window-managers/i3/bumblebee-status/plugins.nix

```diff
@@ -0,0 +1,385 @@
+{ lib
+, pkgs
+, python3Packages
+, ...
+}:
+with pkgs;
+with python3Packages; let
+  mkPluginCfg =
+    { name
+    , requires ? [ ]
+    , requiresUnpackaged ? [ ]
+    , optional ? [ ]
+    , warnings ? [ ]
+    ,
```

Comment:
```suggestion

```

Reply:
`nixpkgs-fmt` did that so I'll just leave it as is